plugins/time-resources/src/components/EditToDo.svelte

```diff
@@ -190,8 +190,8 @@
       </div>
     {/if}
     <div class="slots-content">
-      <div class="flex-row-top justify-between flex-gap-2">
-        <span class="font-medium-14 secondary-textColor">
+      <div class="flex-row-top justify-between items-center flex-gap-2">
+        <span class="font-caps-medium-12 slots-content-title">
```

Comment:
Perhaps we don't need to add an additional style `font-caps-medium-12`, but use two `font-medium-12`and `uppercase`

Reply:
Unfortunately, we have code where we use additional css styles to make the text uppercase. I think it's better to add a complete reusable class, as it's done in design system in Figma. This will help to avoid mistakes in the future and can save a few bytes.